Add issue templates for community plugins (#45)
mattermost/mattermost-marketplace: .github/ISSUE_TEMPLATE/update_plugin.md

---
name: Update plugin
about: Request to update your plugin in the Marketplace
title: "Update $REPOSITORY_NAME to $VERSION"
labels: Plugin/Update
assignees: hanzei
---

<!--

> Thank you very for continuing to develop and maintain your plugin. It will go through a review process to make sure all requirements are still met since the last release.

Reviewer: ```suggestion
Thank you very much for continuing to develop and maintain your plugin. It will go through a review process to make sure all requirements are still met since the last release.
```

Thank you very for continuing to develop and maintain your plugin. It will go through a review process to make sure all requirements are still met since the last release.
-->

#### Summary
<!--
Are there any notable changes since the last release?
-->

> #### Review Commit

Reviewer 2: nit: `Commit` or `Release` (as in the paragraph below)?

Author: `Review Reference`? Then it can be both.

> #### Review Reference

Reviewer: ```suggestion
#### Review reference
```

<!--

> Please link the commit or release that should be used for review.

Reviewer 2: ```suggestion
Please link to an open-source repository and release that should be used for review. It does not suffice to point at an already built plugin: Mattermost code reviews and builds all plugins itself when listing in the marketplace.
```

> Please link to an open-source repository and release or commit that should be used for review. It does not suffice to point at an already built plugin: Mattermost code reviews and builds all plugins itself when listing in the marketplace.

Reviewer: ```suggestion
Please link to an open source repository and release that should be used for review. As Mattermost code reviews and builds all plugins itself when listing in the Marketplace, the link cannot point at an already built plugin.
```

-->

## Checklist
- [ ] [All requirements](https://developers.mattermost.com/extend/plugins/community-plugin-marketplace/#requirements-for-adding-community-plugin-to-the-marketplace) are still met.

> - [ ] The release also has to follow semver. This specifically means for plugins:

Reviewer 3: nit: `s/semver/semantic versioning/`

> - [ ] The release also has to follow [Semantic Versioning](https://semver.org/). This specifically means for plugins:

Reviewer: ```suggestion
- [ ] The release also has to follow [Semantic Versioning](https://semver.org/). For plugins in particular this means:
```

  - If the plugin exposes an API for inter plugin communication, breaking changes to the API require a major version bump.
  - If an update requires manual migration actions from the administrator, a major version bump is required.

> - [ ] A changelog has been published. The link to it is noted via `release_notes_url` in the manifest.

Reviewer 2: :tada:

>   - If an update requires manual migration actions from the administrator, a major version bump is required.

Reviewer: ```suggestion
  - If an update requires manual migration actions from the System Admin, a major version bump is required.
```
